core/src/main/java/com/graphhopper/routing/util/MotorcycleFlagEncoder.java

```diff
@@ -93,30 +100,37 @@ public MotorcycleFlagEncoder( int speedBits, double speedFactor, int maxTurnCost
 
         // autobahn
         defaultSpeedMap.put("motorway", 100);
-        defaultSpeedMap.put("motorway_link", 70);
+        defaultSpeedMap.put("motorway_link", 80);
         defaultSpeedMap.put("motorroad", 90);
         // bundesstraße
         defaultSpeedMap.put("trunk", 80);
         defaultSpeedMap.put("trunk_link", 75);
         // linking bigger town
-        defaultSpeedMap.put("primary", 65);
-        defaultSpeedMap.put("primary_link", 60);
+        defaultSpeedMap.put("primary", 100);
+        defaultSpeedMap.put("primary_link", 80);
         // linking towns + villages
-        defaultSpeedMap.put("secondary", 60);
-        defaultSpeedMap.put("secondary_link", 50);
+        defaultSpeedMap.put("secondary", 90);
+        defaultSpeedMap.put("secondary_link", 70);
         // streets without middle line separation
-        defaultSpeedMap.put("tertiary", 50);
-        defaultSpeedMap.put("tertiary_link", 40);
-        defaultSpeedMap.put("unclassified", 30);
+        defaultSpeedMap.put("tertiary", 80);
+        defaultSpeedMap.put("tertiary_link", 60);
+        defaultSpeedMap.put("unclassified", 40);
         defaultSpeedMap.put("residential", 30);
         // spielstraße
         defaultSpeedMap.put("living_street", 5);
         defaultSpeedMap.put("service", 20);
         // unknown road
-        defaultSpeedMap.put("road", 20);
+        defaultSpeedMap.put("road", 40);
         // forestry stuff
         defaultSpeedMap.put("track", 15);
 
+        enjoyableTags.add("scenic");
```

Comment:
Unfortunately it is not used very often, especially on roads. It's oftimes used on trails and paths http://overpass-turbo.eu/s/cVF. But there are some roads and according to the wiki it should be used for beautiful roads e.g. http://wiki.openstreetmap.org/wiki/Talk:Key:highway#Scenic.2Fhistorical_roads.